udata/core/contact_point/api_fields.py

```diff
@@ -17,6 +17,7 @@
         "owner": fields.Nested(
             user_ref_fields, allow_null=True, description="The user information"
         ),
+        "role": fields.String(description="The role of the contact", required=True),
```

Comment:
Can we add a route to get the roles and their translations ? (in another PR)

Reply:
I created an issue to track it: https://github.com/datagouv/data.gouv.fr/issues/1658